elements/pfe-card/src/pfe-card.html

```diff
@@ -1,6 +1,10 @@
 <!-- pfe-card -->
-<slot class="pfe-card__header" name="pfe-card--header"></slot>
-<slot class="pfe-card__body"></slot>
+<div class="pfe-card__header">
```

Comment:
I think there's a lot of value in moving the classes to be wrapper elements in the shadow DOM so I applied that for the header and footer too!

Reply:
Love it!